Packages/Tracking/Interaction Engine/Runtime/Scripts/UI/Anchors/AnchorableBehaviour.cs

```diff
@@ -684,6 +743,15 @@ private void AttachToNearestAnchor(Anchor preferredAnchor)
                 _preferredAnchor = preferredAnchor;
                 anchor = preferredAnchor;
                 isAttached = true;
+                if (anchor != null && anchor.AttahedHandChirality != ChiralitySelection.NONE)
+                {
+                    _ignorePhysicalHands.HandToIgnore = anchor.AttahedHandChirality;
+                }
+                else if (anchor.AttahedHandChirality != ChiralitySelection.NONE)
+                {
+                    _ignorePhysicalHands.HandToIgnore = ChiralitySelection.NONE;
+
+                }
```

Comment:
To continue with my point about not forcing _ignorePhysicalHands - this should then be wrapped in an `if(_ignorePhysicalHands != null)`